admin/src/app/foms/fom-add-edit/fom-add-edit.component.ts

```diff
@@ -204,23 +203,25 @@ export class FomAddEditComponent implements OnInit, AfterViewInit, OnDestroy {
     });
   }
 
-  async loadForestClients (): Promise<ForestClientResponse[]> {
-   return this.forestSvc.forestClientControllerFind().toPromise()
+async loadForestClients (): Promise<ForestClientResponse[]> {
```

Comment:
Hi Ian, this is really minor. I know we don't use any formatter in FOM, but maybe let's try to keep the indent the same? I feel we need more space here.

Reply:
Indentation adjusted. Thanks for noticing that!